src/server-manager/grpc-gateway.ts

```diff
@@ -394,7 +394,13 @@ export class GrpcGateway extends EventEmitter {
       =====================================================================
       `)
       if (err.code === 14 || err.code === 13 || err.code === 2) {
-        log.silly(PRE, `err code is : ${err.code}, ready to reconnect`)
+        log.info(PRE, `
+        ================================
+        err code : ${err.code},
+        err : ${err},
+        ready to reconnect
+        ================================
```

Comment:
Let's keep using one line log here, don't use multiline logs. Usually multiline logs are error stack. And we need to express more meaningful content here to developers.